package.json

```diff
@@ -130,6 +130,7 @@
 	},
 	"overrides": {
 		"glob-parent@<5.1.2": "5.1.2",
+		"nwsapi@2.2.6": "2.2.5",
```

Comment:
temp fix for broken update with `nwsapi`: https://github.com/dperini/nwsapi/issues/90